Gemfile.lock

```diff
@@ -54,20 +54,20 @@ GEM
     browser_details (0.0.6)
       useragent (>= 0.4)
     builder (3.2.3)
-    bullet (5.7.1)
+    bullet (5.7.2)
```

Comment:
Looks like you somehow updated the dependencies. Can you delete `Gemfile.lock` and run `bundle install` then check in the resulting `Gemfile.lock`?